RevenueCatUI/CustomerCenter/Views/WrongPlatformView.swift

```diff
@@ -92,10 +88,15 @@ struct WrongPlatformView: View {
                     }
                 }
             }
-            if let url = supportURL {
+
+            if let url = supportInformation?.supportURL(localization: localization) {
                 Section {
                     AsyncButton {
-                        openURL(url)
+                        if URLUtilities.canOpenURL(url) {
+                            openURL(url)
+                        } else {
+                            self.showSimulatorAlert = true
```

Comment:
Sorry to get perhaps too nitpicky, but it looks like `URLUtilities.canOpenURL(url)` can return `false` in cases other than the simulator (e.g. extensions or if `application.canOpenURL(url)` return `false`)

Reply:
I've brought back RuntimeUtils, and added that check as well. Only open if `canOpenURL` is true, and its not a sim 😅

Reply:
Ok after discussing the whole approach again with @ajpallares , I think we've got to a solution 😅